app/src/main/res/layout/fragment_arinstructions.xml

```diff
@@ -51,7 +51,7 @@
             android:id="@+id/next_button"
             android:layout_width="wrap_content"
             android:layout_height="wrap_content"
-            android:text="@string/next_step" />
```

Comment:
detta kan man resetta om man vill, precis som min andra liknande kommentar. det är ju bra onödigt att behöva kolla genom 2 filer som är for all intents and purposes samma sak.

spelar ju ingen roll i detta skedet.. men bra att tänka på till framtiden! "git add:a inte sånt som du har ändrat/behöver ändras" tiieep. men orkar du inte resetta så kommer jag inte gråta övereee -- philip triggade mina skrivarfingrar. :shipit: 

nu tog han tom bort sin kommentar.